### ISSUE TITLE
Incorporate latest webidl2.js updates into idlharness

### ISSUE THREAD

**Issue by domenic** [web-platform-tests/wpt#23328]
This is a tracking issue. In particular, I am interested in https://github.com/w3c/webidl2.js/commit/4d22d721fa2cc12ca3058d4be79e26e69c736421 and the not-yet-merged https://github.com/w3c/webidl2.js/pull/466 . Presumably once those are finished there will be a new release of webidl2.js, and we can roll that in.

/cc @saschanaz.

**Comment by saschanaz** [comment]
Just published a new release 👍

**Comment by stephenmcgruer** [comment]
So it looks like we just manually roll `webidl` in a built format only (which is sorta weird). `resources/webidl2/lib/README.md` says:

```
This directory contains a built version of the [webidl2.js library](https://github.com/w3c/webidl2.js).
It is built by running `npm run build-debug` at the root of that repository.
```

That said, I'm not opposed to just doing that procedure again. The last updates were 0d9fecceac869018c2b54b6251600f3d184d98c0, which looks like a cherry-pick of a fix, and before that 10368fa5582b6eeb6ee4556b4a6d0fc7ec896aa4 which brought us to webidl master in late August 2019.

I'm happy to try a roll to master, shouldn't take much effort.
